.github/workflows/analysis.yml

```diff
@@ -50,13 +51,23 @@ jobs:
           -DCMAKE_BUILD_TYPE=Debug
           -DCMAKE_CXX_FLAGS="-Werror -gz -g1"
           -DACTS_BUILD_UNITTESTS=on
+          -DACTS_BUILD_EXAMPLES_UNITTESTS=off
```

Comment:
```suggestion
          -DACTS_BUILD_EXAMPLES_UNITTESTS=OFF
```

Reply:
We mix cases in the options everywhere. One line above it's also lower case. I don't think this is crucial.